scripts/minikube.sh

```diff
@@ -50,26 +50,26 @@ function detect_minikube() {
 
 # install minikube
 function install_minikube() {
-    local mku_version latest_version
-    mku_version=$(${minikube} update-check 2> /dev/null | grep "LatestVersion" || true)
-    latest_version=$(echo "${mku_version}" | cut -d' ' -f2)
-    if [[ -z "${latest_version}" ]]; then
-        # skip: update-check failed for some reason, lets continue with what we have
-        latest_version=${MINIKUBE_VERSION}
+    if [[ "${MINIKUBE_VERSION}" == "latest" ]]; then
+        local mku_version
+        mku_version=$(${minikube} update-check 2> /dev/null | grep "LatestVersion" || true)
+        if [[ -n ${mku_version} ]]; then
```

Comment:
place this variable between `"${quotes}"` please

Reply:
Missed it thanks! done now.